plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -337,6 +343,103 @@ public void testPredicatePushdown()
                 .isFullyPushedDown();
     }
 
+    @Test(dataProvider = "collation")
+    public void testPredicatePushdownWithCollationView(String charset, String collation)
```

Comment:
why separate test with view? To see if JDBC driver can return case-sensitivity for columns of a view?

Reply:
Yes, to be sure that it works. Sometimes if you not able to change collation on table column directly, but want push down. You can make workaround, like create view and change collation on view column and push down will work.